src/main/java/hudson/plugins/view/dashboard/Dashboard.java

```diff
@@ -60,7 +60,6 @@ public boolean isIncludeStdJobList() {
     return includeStdJobList;
   }
 
-  /** @since 2.13 */
```

Comment:
For some reason spotless cannot decide between multiline vs singleline comments:
See these two build
broken: https://ci.jenkins.io/blue/organizations/jenkins/Plugins%2Fdashboard-view-plugin/detail/PR-184/4/pipeline/77#step-91-log-251

```
[2022-07-21T22:25:56.457Z] [ERROR] Failed to execute goal com.diffplug.spotless:spotless-maven-plugin:2.22.8:check (default) on project dashboard-view: The following files had format violations:

[2022-07-21T22:25:56.457Z] [ERROR]     src/main/java/hudson/plugins/view/dashboard/stats/StatSlaves.java
[2022-07-21T22:25:56.457Z] [ERROR]         @@ -14,9 +14,7 @@
[2022-07-21T22:25:56.457Z] [ERROR]          import·org.kohsuke.stapler.export.Exported;
[2022-07-21T22:25:56.457Z] [ERROR]          import·org.kohsuke.stapler.export.ExportedBean;
[2022-07-21T22:25:56.457Z] [ERROR]          
[2022-07-21T22:25:56.457Z] [ERROR]         -/**
[2022-07-21T22:25:56.457Z] [ERROR]         -·*·@author·Lucie·Votypkova
[2022-07-21T22:25:56.457Z] [ERROR]         -·*/
[2022-07-21T22:25:56.457Z] [ERROR]         +/**·@author·Lucie·Votypkova·*/
[2022-07-21T22:25:56.457Z] [ERROR]          public·class·StatSlaves·extends·DashboardPortlet·{
[2022-07-21T22:25:56.457Z] [ERROR]          
[2022-07-21T22:25:56.457Z] [ERROR]          ··@DataBoundConstructor
[2022-07-21T22:25:56.457Z] [ERROR]     src/main/java/hudson/plugins/view/dashboard/Dashboard.java
[2022-07-21T22:25:56.457Z] [ERROR]         @@ -60,9 +60,7 @@
[2022-07-21T22:25:56.457Z] [ERROR]          ····return·includeStdJobList;
[2022-07-21T22:25:56.457Z] [ERROR]          ··}
[2022-07-21T22:25:56.457Z] [ERROR]          
[2022-07-21T22:25:56.457Z] [ERROR]         -··/**
[2022-07-21T22:25:56.457Z] [ERROR]         -···*·@since·2.13
[2022-07-21T22:25:56.457Z] [ERROR]         -···*/
[2022-07-21T22:25:56.457Z] [ERROR]         +··/**·@since·2.13·*/
[2022-07-21T22:25:56.715Z] [ERROR]          ··@DataBoundSetter
[2022-07-21T22:25:56.715Z] [ERROR]          ··public·void·setIncludeStdJobList(boolean·includeStdJobList)·{
[2022-07-21T22:25:56.715Z] [ERROR]          ····this.includeStdJobList·=·includeStdJobList;
[2022-07-21T22:25:56.715Z] [ERROR]     src/main/java/hudson/plugins/view/dashboard/DashboardLog.java
[2022-07-21T22:25:56.715Z] [ERROR]         @@ -3,9 +3,7 @@
[2022-07-21T22:25:56.715Z] [ERROR]          import·java.util.logging.Level;
[2022-07-21T22:25:56.715Z] [ERROR]          import·java.util.logging.Logger;
[2022-07-21T22:25:56.715Z] [ERROR]          
[2022-07-21T22:25:56.715Z] [ERROR]         -/**
[2022-07-21T22:25:56.715Z] [ERROR]         -·*·@author·Marco·Ambu
[2022-07-21T22:25:56.715Z] [ERROR]         -·*/
[2022-07-21T22:25:56.715Z] [ERROR]         +/**·@author·Marco·Ambu·*/
[2022-07-21T22:25:56.715Z] [ERROR]          public·class·DashboardLog·{
[2022-07-21T22:25:56.715Z] [ERROR]          
[2022-07-21T22:25:56.715Z] [ERROR]          ··private·static·String·loggerName·=·"DashboardView";
[2022-07-21T22:25:56.715Z] [ERROR] Run 'mvn spotless:apply' to fix these violations.
```

fixed in 52ceb57 by removing comments and green build: https://ci.jenkins.io/job/Plugins/job/dashboard-view-plugin/job/PR-184/5/

Reply:
It's probably because Spotless uses different versions of the Google-Java-Format when building with different Java versions. I'll just remove the step on master for now...